sdk/formrecognizer/ai-form-recognizer/src/constants.ts

```diff
@@ -1,11 +1,23 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+/**
+ * Defines known cloud audiences for Form Recognizer.
+ */
+export enum KnownFormRecognizerAudience {
+  /** Azure China */
+  AzureResourceManagerChina = "https://cognitiveservices.azure.cn/.default",
```

Comment:
why is there ARM in the name?

Reply:
![image](https://user-images.githubusercontent.com/10452642/156086046-16f2e0d3-14f0-473b-8bd1-ba87e77cf0c2.png)
Container registry wasn't exactly arm, but also had the same name.

I just went with consistency, is there a better name?

Reply:
`https://management.chinacloudapi.cn` sounds like an ARM scope but `https://cognitiveservices.azure.cn` is not. Perhaps use cognitive instead, e.g. `CognitiveChina`?

Reply:
Right, we use ARM in the name because those are management audiences that are used by the service team.

Reply:
`KnownFormRecognizerAudience.AzureCognitiveServicesChina` ?

Reply:
BTW, Java also followed "resource manager" convention in the naming.
![image](https://user-images.githubusercontent.com/10452642/156089025-f55e88e0-16a0-427a-9396-7512cd440dbf.png)
https://github.com/samvaity/azure-sdk-for-java/blob/e0d055cbe7f575c5ba004d1b07a9a48487cf6f6e/sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/models/FormRecognizerAudience.java#L11-L19

/cc @samvaity

Reply:
By architect recommendation, .NET is using:

```csharp
public static DocumentAnalysisAudience AzureChina { get; } = new DocumentAnalysisAudience(AzureChinaValue);
public static DocumentAnalysisAudience AzureGovernment { get; } = new DocumentAnalysisAudience(AzureGovernmentValue);
public static DocumentAnalysisAudience AzurePublicCloud { get; } = new DocumentAnalysisAudience(AzurePublicCloudValue);
```

This aligns better with the naming in the Azure portal and documentation, as well as the [Azure.Identity::AzureAuthorityHosts](https://apiview.dev/Assemblies/Review/6a40db7f9eeb41dabfe5768f1e796c81#Azure.Identity.AzureAuthorityHosts).   The ACR values are named that way specifically because they are using the ARM audience and plan on adding ARC-specific values.  

For more context, please see the discussion [here](https://github.com/Azure/azure-sdk-for-net/pull/26583#discussion_r791962593)

Reply:
Thanks for the comments. Going with this.

`KnownFormRecognizerAudience`
- `AzureChina`
- `AzureGovernment`
- `AzurePublicCloud`

Reply:
Can we drop the "Known"? Doesn't add anything IMO.

Reply:
@witemple-msft having Known in the type name is a pattern the code generator is using for some time now (and transitively, all auto-generated clients) but I agree that it should be dropped. I understand this type is generated but does not get used to type anything in the public surface. Instead related places are typed as string so there should not be a name collision concern. @joheredi do you remember why we have that `Known` prefix?